src/form/Form.tsx

```diff
@@ -256,6 +257,9 @@ function Form({
     initState.renderCallbacks[formKey] = () => {
       setRender((render) => !render);
     };
+    // Need to initialize setAuthId somehow
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const test = new Client(null, null, setAuthId);
```

Comment:
thinking i should probably export an object from Client that contains setAuthId and then I can set it here, so that I don't need to have a contrived constructor call

doesn't seem appropriate to put into initState semantically, though it would fit the global access pattern